app/app/javascript/types/Argyle.d.ts

```diff
@@ -38,5 +39,9 @@ type ArgyeUIEvent = {
     userId: string
     accountId?: string
     itemId?: string
+    errorCode?: string
```

Comment:
Per Argyle docs, this should probably be an enum

```suggestion
errorCode?: 'auth_required' | 'connection_unavailable' | 'expired_credentials' | 'invalid_auth' | 'invalid_credentials' | 'mfa_cancelled_by_the_user'
```